js/reducers/tests/sources.js

```diff
@@ -1,3 +1,7 @@
+/* eslint brace-style: 0 */
+/* globals equal */
+/* eslint camelcase: 0 */
+/* eslint no-unused-vars: 0 */
```

Comment:
It'd be really nice to have an `.eslint` in all test directories that automatically define the globals so that we don't need this at the top of every test file.